delfin/api/schemas/access_info.py

```diff
@@ -17,10 +17,36 @@
 update = {
     'type': 'object',
     'properties': {
-        'host': parameter_types.hostname_or_ip_address,
-        'port': parameter_types.tcp_udp_port,
-        'username': {'type': 'string', 'minLength': 1, 'maxLength': 255},
-        'password': {'type': 'string', 'minLength': 1, 'maxLength': 255},
+        'rest_access': {
```

Comment:
Suggest to make it just 'rest' and 'ssh' . No need to append "_access" . What do you think?

Reply:
Thanks, have changed.